src/main/java/net/dv8tion/jda/core/entities/User.java

```diff
@@ -111,6 +113,14 @@
     RestAction<PrivateChannel> openPrivateChannel();
 
     /**
+     * Finds and collects all {@link net.dv8tion.jda.core.entities.Guild Guild} instances that contain this {@link net.dv8tion.jda.core.entities.User User} within the current {@link net.dv8tion.jda.core.JDA JDA} instance.<br>
+     * <p>This method is a shortcut for {@link net.dv8tion.jda.core.JDA#getMutualGuilds(User...) JDA.getMutualGuilds(User)}.</p>
```

Comment:
Paragraph doesn't *need* to be closed for future reference.
The start of a new paragraph *should* be separate from the previous paragraph by an extra line of space in the javadoc comment.